packages/url-loader/tests/lib/cloudinary.spec.js

```diff
@@ -182,70 +182,53 @@ describe('Cloudinary', () => {
 
     describe('cropping, resizing', () => {
 
-      it('should create a Cloudinary URL with widthResize smaller than width', () => {
+      it('should create a Cloudinary URL with a width, height, and custom crop', () => {
```

Comment:
suggestion: this is a rather general suggestion but have you maybe thought about writing spec files in .ts instead of .js?

We are using it in both my work and personal/community projects and it has proven to be quite useful :)

Reply:
havent yet. happy to look into it outside of this scope, is the implication simply that you get autocomplete and such? id be worried of having to potentially support complex types for writing the tests themselves, is that the case?

Reply:
started a discussion so it doesnt getl ost: https://github.com/colbyfayock/cloudinary-util/discussions/141